### PR TITLE
[14.0] [ADD] account statement import online ing

### PR DESCRIPTION
**Pull request by NL66278**
Clear payment_ref of data also in other fields.

### CODE REVIEW

**Comment by NL66278** [comment]
Includes changes in https://github.com/OCA/bank-statement-import/pull/431

**Review by StefanRijnhart — CHANGES_REQUESTED** [review]
Looking good in principle, but can you rebase now that https://github.com/OCA/bank-statement-import/pull/431 is merged? The commit specific to this PR looks strange, as if it reverses the refactoring after the `if response.status_code == 200:` line.

**Comment by NL66278** [comment]
@StefanRijnhart  rebased and then manually reset the remaining unwanted change at the location you indicated.

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

File: account_statement_import_online_ponto_ing/tests/test_account_statement_import_online_ponto.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+_module_ns = "odoo.addons.account_statement_import_online_ponto"
+_provider_class = (
+    _module_ns
+    + ".models.online_bank_statement_provider_ponto"
+    + ".OnlineBankStatementProviderPonto"
+)
+
+
+class TestAccountBankAccountStatementImportOnlineQonto(common.TransactionCase):
```
Comment (alexey-pelykh) [inline account_statement_import_online_ponto_ing/tests/test_account_statement_import_online_ponto.py]: Qonto <> Ponto

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

**Comment by alexey-pelykh** [comment]
Except one typo code is LGTM

**Comment by NL66278** [comment]
@alexey-pelykh I corrected it Thanks!

**Review by alexey-pelykh — APPROVED** [review]

**Pull request merged**